repo-template.tf

```diff
@@ -0,0 +1,34 @@
+resource "github_repository" "dataworks_repo_template" {
+  name        = "dataworks_repo_template"
```

Comment:
Can the names be hyphens instead of underscores like all the others?

Reply:
Done.